git-clang-format

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python
```

Comment:
Would you be willing to modify the commit message to specify exactly where this came from (including the hash)?  Then I can double-check that it matches.

Reply:
I _think_ this is the same file but has suffered `black` formatting?

I specifically put `exclude: git-clang-format` in the `.pre-commit-config.yaml` in my previous patch to avoid this being affected by our `pre-commit` hooks in the project. This will allow simplier tracking of the upstream `git-clang-format`.

I think this patch chain needs to be rebased on the current project `HEAD`, this file reverted and `pre-commit run --all-files` ran.

As an aside: you may want to use [`git subtree`](https://www.atlassian.com/git/tutorials/git-subtree) to pull the upstream [`git-format-clang`](https://github.com/llvm/llvm-project/blob/main/clang/tools/clang-format/git-clang-format) in. It is _really_ easy to track where and which version of the upstream file we have pulled in then and doesn't suffer from the initialisation worries and pain of a `git submodule`

Reply:
Yes, the file got changed due to black formatting. To avoid confusion I will have the revert the formatting changes

Reply:
This is the same file and there is no change. You can now see the git-clang-format file same as before as the black formatting changes are reverted.